.github/actions/deploy/action.yaml

```diff
@@ -83,6 +83,11 @@ runs:
       shell: bash
       run: sh tests/smoke/deploy-sorbet-resources.sh end2end
       working-directory: ./.github/scripts/end2end/operator
+    - name: Deploy metadata
+      shell: bash
+      run: bash deploy-metadata.sh
+      working-directory: ./.github/scripts/end2end
+      if: ${{ env.ENABLE_RING_TESTS == 'true' }}
```

Comment:
better to have it as an `input` rather than an env variable
(the input parameter may however takes its "default" value from env)